app/views/includes/head.jade

```diff
@@ -34,6 +34,9 @@ head
 
   link(rel='stylesheet', href='https://fonts.googleapis.com/icon?family=Material+Icons')
 
+  //font awesome CDN
+  script(src="https://use.fontawesome.com/3c82830ea0.js")
```

Comment:
What's this for? Is this not enough https://fonts.googleapis.com/icon?family=Material+Icons